popt.3

```diff
@@ -246,12 +246,15 @@ Option callbacks should match the following prototype:
 .sp
 .nf
 .BI "void poptCallbackType(poptContext con, 
+.BI "                      enum poptCallbackReason reason,
 .BI "                      const struct poptOption * opt, 
-.BI "                      const char * arg, void * data);
+.BI "                      const char * arg, const void * data);
 .fi
 .sp
 The first parameter is the context which is being parsed (see the next
-section for information on contexts), \fIopt\fR points to the option
+section for information on contexts). \fIreason\fR is
+POPT_CALLBACK_REASON_PRE, POPT_CALLBACK_REASON_POST, or
+POPT_CALLBACK_REASON_OPTION. \fIopt\fR points to the option
```

Comment:
Shouldn't these callback constants be wrapped in `\fB` tags?

Reply:
Fixed up.